packages/components/src/button/docs/Button.stories.mdx

```diff
@@ -34,6 +34,28 @@ import { Text } from "@components/typography";
     githubPath="/packages/components/src/button/src"
 />
 
+## Guidelines
+
+### When to use
+
+- To trigger an action such as submitting a form, exporting data.
+
+### Content
+
+Buttons should:
+
+- Use strong, actionable verbs.
+- Be positionned in consistent locations.
+
+### Accessibility
+
+- Use the disabled prop to set the disabled state of the button.
```

Comment:
Is it really an accessibility tip? It seems quite obvious to use a disabled prop to disabled a button

Reply:
It's indeed probably too obvious, I was trying to avoid not to have accessibility tips.

Reply:
Could you write something about the tooltips here then? It makes sense to me to mention that here